README.md

```diff
@@ -100,7 +96,11 @@ The following variants are no longer supported:
 * `aws-k8s-1.18`
 * `aws-k8s-1.19`
 * `aws-k8s-1.20`
+* `aws-k8s-1.21`
+* `aws-k8s-1.21-nvidia`
+* `metal-k8s-1.21`
 * `vmware-k8s-1.20`
+* `vmware-k8s-1.21`
```

Comment:
This list of EOL variants is getting pretty long and is gonna get exponentially longer since each K8s version are associated with 4 different variants. Is there any value in maintaining this list? Maybe just keep variants associated with the 3 most recent EOL'd K8s versions?

Reply:
I agree. I think it may make more sense to just state something like "All kubernetes variants using `1.21` and earlier". Would that make sense? Then we just update that last version number each time we drop a version.

Reply:
Updated.